pkg/server/server.go

```diff
@@ -76,9 +76,9 @@ var immudbTextLogo = " _                               _ _     \n" +
 
 // Initialize initializes dependencies, set up multi database capabilities and stats
 func (s *ImmuServer) Initialize() error {
-	s.Logger.Infof("%s\n%s\n%s\n\n", immudbTextLogo, version.VersionStr(), s.Options)
-
-	if s.Options.Logfile != "" {
+	if s.Options.IsJSONLogger() {
```

Comment:
@jeroiraz does this satisfy the condition which you were mentioning?

- json format into stdout, messages are not printed (logo and CLIs messages)
- text format into stdout, messages are printed into stdout

Reply:
I think so. The same logic may be applied in the logo

Reply:
The logo is only printed in case of stdout to file